### ISSUE TITLE
Migration files and merge conflicts

### ISSUE THREAD

**Issue by SamyPesse** [drizzle-team/drizzle-kit-mirror#160]
Currently, when using `drizzle-kit` and storing the migration files in Git, handling merge conflicts is currently very very hard.

The main issues is that Drizzle doesn't use an approach where it avoids conflicts (like `changeset` does, or similar tools).

It'd be great if there was a command to resolve merge conflicts in migration files.

**Comment by fabis94** [comment]
Yeah honestly it's bizarre that there's no resources or CLIs for dealing with this issue. It's pretty much guaranteed that anyone that uses drizzle in a repo with multiple people working on it is going to run into this.
